src/main/java/com/revature/models/ResultSummary.java

```diff
@@ -1,14 +1,23 @@
 package com.revature.models;
 
 import java.net.URI;
+import javax.persistence.Entity;
+import javax.persistence.GeneratedValue;
+import javax.persistence.GenerationType;
+import javax.persistence.Id;
 
 import com.revature.responsecollector.JMeterResponseCollector;
 import lombok.EqualsAndHashCode;
-import lombok.Getter;
-import lombok.Setter;
 
-@Getter @Setter @EqualsAndHashCode
+
+@EqualsAndHashCode
```

Comment:
This class could use some more Lombok annotations instead of generating them with Eclipse. `@Data` could be used, which is a combination of `@NoArgsConstructor`, `@AllArgsConstructor`, `@Getter`, `@Setter`, `@EqualsAndHashCode`, and `@ToString`. We could also add in an `@RequiredArgsConstructor` and configure it to only include the parameters that you need.